app/v1/validation/GroupRatioValidator.scala

```diff
@@ -34,17 +34,9 @@ trait GroupRatioValidator extends BaseValidation {
     }
   }
 
-  private def validateGroupEBITDA(implicit path: JsPath): ValidationResult[Option[Boolean]] = {
-    (groupRatioModel.isElected,  groupRatioModel.groupEBITDAChargeableGains.isDefined) match {
-      case (false, true) => GroupEBITDASupplied(groupRatioModel.groupEBITDAChargeableGains).invalidNec
-      case (true, false) => GroupEBITDANotSupplied().invalidNec
-      case _ => groupRatioModel.groupEBITDAChargeableGains.validNec
-    }
-  }
-
   def validate(implicit path: JsPath): ValidationResult[GroupRatioModel] =
     (validateGroupRatioElected,
-      validateGroupEBITDA,
+      groupRatioModel.groupEBITDAChargeableGains.validNec,
```

Comment:
We can remove this line now - you'll just have to remove one of the wildcards from the `mapN` beneath